spec/puppet-syntax/manifests_spec.rb

```diff
@@ -107,20 +122,43 @@
           expect(has_errors).to eq(false)
         end
       end
+    when 4.0..4.99
```

Comment:
Thanks @trlinkin, this looks better. Could you please move this `when` block further up so that the version numbers decrement sequentially in order?

If you could also squash this commit into the original relevant commit you made, that'd be great.